source/_integrations/esphome.markdown

```diff
@@ -29,6 +29,8 @@ ha_platforms:
   - switch
 ha_integration_type: integration
 ha_dhcp: true
+works_with: 
+  - local
```

Comment:
According to our Partner program, ESPHome doesn't live up to be allowed into this?

> Integration Quality Scale requirement of Gold or better

ESPHome doesn't meet this requirement.